VideoWeb/VideoWeb.Common/Caching/DistributedJOHConsultationRoomLockCache.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Caching.Distributed;
+
+namespace VideoWeb.Common.Caching
+{
+    public class DistributedJOHConsultationRoomLockCache : RedisCacheBase<string , bool>,IDistributedJOHConsultationRoomLockCache
+    {
+        public DistributedJOHConsultationRoomLockCache(IDistributedCache distributedCache) : base(distributedCache)
+        {
+            CacheEntryOptions = new DistributedCacheEntryOptions
+            {
+                SlidingExpiration = TimeSpan.FromMinutes(5) //reduce this to few seconds?
```

Comment:
I dont think the lock needs to last 5 minutes. It should only be very temporary, like 5 seconds then expire